vars.tf

```diff
@@ -114,3 +118,43 @@ variable "create_dashboard" {
   type        = bool
   default     = true
 }
+
+
+/*
+ * ECS and ASG configuration
+ */
+
+variable "asg_min_size" {
+  default = "1"
+}
+
+variable "asg_max_size" {
+  default = "5"
+}
+
+variable "alarm_actions_enabled" {
+  default     = false
+  description = "True/false enable auto-scaling events and actions"
+}
+
+variable "ssh_key_name" {
+  default = ""
+}
+
+/*
+ * VPC configuration
+ */
+
+variable "aws_zones" {
+  type    = list(string)
+  default = ["us-east-1c", "us-east-1d", "us-east-1e"]
```

Comment:
Do we want these to be us-east-2*?

Reply:
Probably not yet. Eventually we may just remove the default.